lib/memory.c

```diff
@@ -271,15 +271,24 @@ static void *alloc0(size_t size)
 
         /* 9% of logs in m0trace log file is due to M0_ENTRY  and M0_LEAVE */
 	//M0_ENTRY("size=%zi", size);
+#ifdef ENABLE_DEV_MODE
```

Comment:
Why is this needed? M0_FI_ENABLED* is false outside of DEV_MODE.

Reply:
M0_ENTRY("size=%zi", size); was completely disabled in one of the previous patch, during debug enabled build, this is good to have message so added under DEV_MODE flag.

Reply:
Added only M0_ENTRY under dev mode

Reply:
I still do not understand. In the current main this trace call is enabled all the time. Why do you want to eliminate it in non-dev builds?